main.go

```diff
@@ -148,6 +159,12 @@ func main() {
 	log.Debugf("PATH: %s", os.Getenv("PATH"))
 
 	if cfg.IsDebug {
+		flutterBinPath, err := exec.LookPath("flutter")
```

Comment:
it is already done here: https://github.com/bitrise-steplib/bitrise-step-flutter-installer/pull/10/files#diff-7ddfb3e035b42cd70649cc33393fe32cR67

Reply:
The new flutter version is installed inbetween.